src/config.ts

```diff
@@ -22,14 +22,17 @@ export interface UserConfig {
   theme?: string;
 
   /**
-   * The way documentable items (contracts, functions, etc.) will be organized
-   * in pages. Built in options are: 'single' for all items in one page, and
-   * 'items' for one page per item. More customization is possible by defining
-   * a function that returns a page path given the AST node for the item and
-   * the source unit where it is defined.
+   * The way documentable items (contracts, functions, custom errors, etc.)
+   * will be organized in pages. Built in options are:
+   * - 'single': all items in one page
+   * - 'items': one page per item
+   * - 'files': one page per input Solidity file
+   * More customization is possible by defining a function that returns a page
+   * path given the AST node for the item and the source unit where it is
+   * defined.
    * Defaults to 'single'.
    */
-  pages?: 'single' | 'items' | ((item: DocItem, file: SourceUnit) => string | undefined);
+  pages?: 'single' | 'items' | 'files' | PageAssigner;
```

Comment:
Didn't this remove the ability to provide a custom organization?  Why would we take this away from users?

Reply:
`PageAssigner` is an alias for the same function type as before.

Reply:
Oh, I missed that. All good.